### PR TITLE
Added recipe for conda-workon

### PR DESCRIPTION
**Pull request by pmlandwehr**
Pinging @flub in case they'd like to be added as a maintainer to the `conda-workon` builder on [conda-forge](http://conda-forge.github.io), a library of community-built [`conda`](http://conda.pydata.org) packages. The maintenance burden here is light: all testing and releases are built and deployed automatically with CI. Changes to how the package is being built can be controlled by changing the recipe which will be located at https://github.com/conda-forge/conda-workon-feedstock shortly after this PR is finished and merged. If you aren't interested in helping maintain the build, that's entirely fine too.

Also, if you could let us know which BSD license you've put `conda-workon` out under, that'd be great (2-Clause or 3-Clause). Ditto if you have time to push version `0.4` to pypi at some point… :D

### CODE REVIEW

File: recipes/conda-workon/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    - conda
+
+test:
+  commands:
+    - conda workon --help
+    - conda-workon --help
+
+about:
+  home: https://bitbucket.org/flub/conda-workon
+  license: BSD (Clause unspecified)
```
Comment (jakirkham) [inline recipes/conda-workon/meta.yaml]: Could we please raise an issue to get this specified?

File: recipes/conda-workon/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  build:
+    - python
+    - setuptools
+    - conda
+
+  run:
+    - python
+    - conda
+
+test:
```
Comment (jakirkham) [inline recipes/conda-workon/meta.yaml]: Import tests?

Comment (pmlandwehr) [inline recipes/conda-workon/meta.yaml]: Ah, whoops.

File: recipes/conda-workon/meta.yaml (hunk 1 above)
Comment (pmlandwehr) [inline recipes/conda-workon/meta.yaml]: Yeah - mentioned in ping, but will add that at the repo's home.

Comment (pmlandwehr) [inline recipes/conda-workon/meta.yaml]: See [here](https://bitbucket.org/flub/conda-workon/issues/3/type-of-bsd-license)

Comment (jakirkham) [inline recipes/conda-workon/meta.yaml]: Thanks.

**Comment by flub** [comment]
Hi, sure you can add me as maintainer and I'll keep it updated on new releases - I've always meant to make conda packages too but never got round to it.

The license was meant to be the same as conda itself, so 3-clause.  Pretty shocking I don't have a LICENSE file though, will fix that in next release too.

**Comment by jakirkham** [comment]
Thanks @flub. Please also let us know if there are any changes you would recommend for the recipe.

File: recipes/conda-workon/meta.yaml (hunk 1 above)
Comment (jakirkham) [inline recipes/conda-workon/meta.yaml]: Sounds like this should be `BSD 3-Clause`.

File: recipes/conda-workon/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    - conda-workon --help
+
+about:
+  home: https://bitbucket.org/flub/conda-workon
+  license: BSD (Clause unspecified)
+  summary: 'Activate conda environments in subshells'
+
+extra:
+  recipe-maintainers:
+    - pmlandwehr
```
Comment (jakirkham) [inline recipes/conda-workon/meta.yaml]: Also please add @flub.

File: recipes/conda-workon/meta.yaml (hunk 2 above)
Comment (jakirkham) [inline recipes/conda-workon/meta.yaml]: Done below.

**Comment by jakirkham** [comment]
Weird. This has merge conflicts. 😕 Closing and reopening to see if that fixes it.

**Comment by jakirkham** [comment]
Alright going to merge this manually.

**Comment by jakirkham** [comment]
Thanks @pmlandwehr and @flub.

**Comment by jakirkham** [comment]
As this is you first time signing up as a maintainer, @flub, I would like to explain what is about to happen. When this recipe is merged, a job will run on Travis CI to automatically convert this recipe into a feedstock (repo). The feedstock will begin its first build once generated and release packages to the conda-forge channel. In that process, you should get an invitation to join conda-forge. Once accepted, you will be given permissions to this feedstock (and any others you are listed as a maintainer on). 

Regarding making changes to the recipe, please proceed with the normal workflow one would with any repository. Please fork the repository, make changes to your fork, and PR them back. This is important to follow exactly as all branches on the original repository will automatically deploy anything they build.

If you have and questions or concerns, please let us know. Welcome to conda-forge. 😄

**Pull request merged**